kickstarts/partials/post/ui_compile.ks.erb

```diff
@@ -1,4 +1,4 @@
-npm install -g gulp bower
+npm install -g gulp bower yarn
 
 pushd /var/www/miq/vmdb
   bower -F --allow-root install
```

Comment:
@AllenBW No longer need to run bower in 'vmdb' either?  Just confirming as bin/setup and bin/update are still calling it...